tests/bud.bats

```diff
@@ -5738,8 +5738,8 @@ _EOF
   run_buildah build -t buildkitbase $WITH_POLICY_JSON -f $contextdir/Dockerfilebuildkitbase $contextdir/
 
   # try reading something from persistent cache in a different build
-  run_buildah 125 build -t testbud $WITH_POLICY_JSON -f $contextdir/Dockerfilecachefromimage
-  expect_output --substring "no stage found with name buildkitbase"
+  run_buildah 125 build -t testbud $WITH_POLICY_JSON -f ${TEST_SCRATCH_DIR}/bud/buildkit-mount-from/Dockerfilecachefromimage
+  expect_output --substring "no stage or additional build context found with name buildkitbase"
   run_buildah rmi -f buildkitbase
 }
 
```

Comment:
This is a fix for CVE-2024-9675, I'm not sure why it's here for CVE-2024-11218.  Plus the comment seems to be talking mostly about adding a test for that CVE and there's  a lot more than tests here.